icons/readme.md

```diff
@@ -0,0 +1,14 @@
+# Project and other icons
+
+In this folder a library of SVGs and (derived) Windows .ico files offers users a choice of simple graphical images to use as icons, for example to represent processes and resources in your file system.
+
+At user discretion, any of these could be associated with a script or a file type.
+
+For example, on Windows it is straightfoward to create a shortcut for a process script such as [../xp3.bat](../xp3.bat), copy the shortcut to any convenient location, and change its icon to provide a visual association for the operation. The process can be invoked by double-clicking on it or (in the case of processes that expect runtime arguments) by dragging a resource to it. 
+
+In an effort to support [XProc 3.0](https://xproc.org) branding and awareness efforts, these images embed Kanava, the piscine [XProc mascot](https://xproc.org/img/logo.png), with the kind permission of his author, Bethan Tovey-Walsh.
```

Comment:
What was/is the original license for logo? Did you draw the related SVG files completely from scratch or derived those files in this PR from the linked PNG or other assets from the XProc site? If the latter and they do have a license, Norm gave permission for that?

Reply:
I discussed with Wendell that the original author of the images gave permission with their copyright for this express purpose, so I will resolve this comment and re-review.